Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/WorkflowBusiness.java

```diff
@@ -113,6 +115,31 @@ public WorkflowBusiness() {
         }
     }
 
+    private String selectEngineEndpoint(String applicationClass) throws BusinessException {
+        long min = 100000;
+        long tmp;
```

Comment:
Variable could be declared at line 126. Could be renamed to, e.g. "runningWorkflows".